mppsolar/outputs/prom.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import re
+
+from .baseoutput import baseoutput
+from ..helpers import get_kwargs, key_wanted
+
+log = logging.getLogger("prom")
+
+
+class prom(baseoutput):
+    def __str__(self):
+        return "outputs Node exporter prometheus format to standard out"
+
+    def __init__(self, *args, **kwargs) -> None:
+        log.debug(f"processor.value __init__ kwargs {kwargs}")
+
+    def printHeader(command, description):
+        pass
+
+    def output(self, *args, **kwargs):
+        log.info("Using output processor: value")
+        log.debug(f"kwargs {kwargs}")
+        tag = get_kwargs(kwargs, "tag")
+        dev = get_kwargs(kwargs, "name")
+        data = get_kwargs(kwargs, "data")
+        name = get_kwargs(kwargs, "name")
+        if data is None:
+            return
+
+        # check if config supplied
+        config = get_kwargs(kwargs, "config")
+        if config is not None:
+            log.debug(f"config: {config}")
+            # get formatting info
+            remove_spaces = config.get("remove_spaces", True)
+            keep_case = config.get("keep_case", False)
+            filter = config.get("filter", None)
+            excl_filter = config.get("excl_filter", None)
+            tag = config.get("tag", None)
+            name = cconfig.get("name", "mpp_solar")
```

Comment:
seems like a typo

Reply:
Yes it is and I have corrected. Also per best practice for Prometheus I have changed the output. uploading shortly.